ui-common/pom.xml

```diff
@@ -157,6 +157,13 @@
 			<version>2.0.05</version>
 			<type>jar</type>
 		</dependency>
+		<dependency>
+			<groupId>javax.jnlp</groupId>
+			<artifactId>jnlp</artifactId>
+			<version>1.6</version>
+			<scope>system</scope>
+			<systemPath>/usr/share/icedtea-web/javaws.jar</systemPath>

```

Comment:
This systemPath probably should be a property ${local.jnlp.path} rather than hard coded to /usr/share/icedtea-web

Reply:
okay, that's fine, I will fix it